### ISSUE TITLE
Models using _id and id should be using only _id

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazi#905]
Previously couch subdocuments had no _id, so there is an id-generation process that stores an ID for each subdocument.

With Mongo, that is done automatically, so models and processes depending upon old ID should be migrated to just use _id.

**Comment by txau** [comment]
@RafaPolit is this still a thing?

**Comment by RafaPolit** [comment]
I believe it is, at least for template properties or thesauri items.  We need to check.

**Comment by txau** [comment]
@RafaPolit you are right, I just checked and botth template properties and thesauri items still use id.

**Comment by RafaPolit** [comment]
Thesauri, relationshiptypes (including route validation and model) still use id.  We should create issues to remove this.

**Comment by fnocetti** [comment]
At this point we think the only place is the thesauri values.
